middleware/getUserInfo.js

```diff
@@ -2,22 +2,36 @@ const axios = require("axios");
 
 function getUserInfo(req, res, next) {
   const userData = {};
+  const repoList = [];
   // TODO: issue #47 get user info from database
   // will need join between repos and user information
 
   // TODO: issue #46 get user repos from github
 
   const userName = req.user.ghUsername;
 
-  axios.get("https://api.github.com/users/" + userName).then(response => {
-    userData.profileImg = response.data.avatar_url;
-    userData.about = response.data.bio;
-  });
-
-  // TODO: issue #45 get user info from github
+  axios
+    .get("https://api.github.com/users/" + userName)
+    .then(response => {
+      userData.name = response.data.name;
+      userData.profileImg = response.data.avatar_url;
+      userData.about = response.data.bio;
+      return axios.get("https://api.github.com/users/" + userName + "/repos");
```

Comment:
Looks good just one small thing, lets move this second http request out into a separate function.  Its best to have a function do one thing only.  Have one function to get user info and one function to get repos.  If you have questions we can take a look at it in the morning.